ts/features/bonus/bpd/analytics/index.ts

```diff
@@ -52,6 +67,68 @@ const trackAction = (mp: NonNullable<typeof mixpanel>) => (
       return mp.track(action.type, { ibanStatus: action.payload.status });
     case getType(bpdUpsertIban.failure):
       return mp.track(action.type, { reason: action.payload.message });
+
+    // transactions
+    case getType(bpdTransactionsLoad.request):
+      return mp.track(action.type, { awardPeriodId: action.payload });
+    case getType(bpdTransactionsLoad.success):
+      return mp.track(action.type, {
+        awardPeriodId: action.payload.awardPeriodId,
+        hashPan: action.payload.results.map(r => r.hashPan),
+        idTrxAcquirer: action.payload.results.map(r => r.idTrxAcquirer),
+        idTrxIssuer: action.payload.results.map(r => r.idTrxIssuer),
+        trxDate: action.payload.results.map(r => r.trxDate),
+        circuitType: action.payload.results.map(r => r.circuitType)
+      });
+    case getType(bpdTransactionsLoad.failure):
+      return mp.track(action.type, {
+        awardPeriodId: action.payload.awardPeriodId,
+        reason: action.payload.error.message
+      });
+
+    // CashBack details
+    case getType(bpdDetailsLoadAll):
+    case getType(bpdLoadActivationStatus.request):
+      return mp.track(action.type);
+    case getType(bpdLoadActivationStatus.success):
+      return mp.track(action.type, {
+        enabled: action.payload.enabled,
+        payoffInstr: action.payload.payoffInstr
```

Comment:
```suggestion
```
This is the IBAN (yes I know, its name is far away from that meaning).
We have to remove it from tracking